### ISSUE TITLE
[Text, IOS]String with newline breaks the alignment

### ISSUE THREAD

**Issue by mk0116** [wix/react-native-ui-lib#2457]
## Description

[Text]String with newline breaks the alignment.

When I add new line character('\n') in the text and nest a Text, it breaks the alignment.
The first line should be aligned center but left.
This only happens on iOS.
It works fine on Android.

### Related to

- [v] Components
- [ ] Demo
- [ ] Docs
- [ ] Typings

### Code snippet

```
<Text
    textColor
    center>
    <Text bold>test</Text>
    {'\nrest string'}
</Text>
```

### Screenshots/Video

<img width="255" alt="스크린샷 2023-02-10 오전 9 58 44" src="https://user-images.githubusercontent.com/11452913/217973363-7c138b8c-7b56-4ecf-91c8-eac051762389.png">

### Environment

- React Native version: 0.70.3
- React Native UI Lib version: 6.29.0

### Affected platforms

- [ ] Android
- [v] iOS
- [ ] Web

**Comment by mk0116** [comment]
This has not been fixed yet.

**Comment by Inbal-Tish** [comment]
@mk0116 Hi. What result do you expect to get?

**Comment by mk0116** [comment]
Hello,

I expect all elements within the parent Text component to be aligned
centrally.

However, it is evident that the first line is aligning to the left instead
of the center.

Interestingly, when I use the standard Text component, the alignment is
correctly set to the center.

Additionally, I have been unable to figure out a method to align the nested
text components with react-native-ui-lib

Thank you.




2023년 7월 10일 (월) 오후 3:59, Inbal Tish ***@***.***>님이 작성:

**Comment by mk0116** [comment]
Thank you so much!
Because of that I stopped my migration. That will help me a lot.


2023년 7월 10일 (월) 오후 9:34, Inbal Tish ***@***.***>님이 작성:

**Comment by Inbal-Tish** [comment]
@mk0116 Hi. Issue solved on version 7.6.0. Thank you for reporting!

**Comment by mk0116** [comment]
Thank you!!!

2023년 7월 24일 (월) 오후 5:02, Inbal Tish ***@***.***>님이 작성:
